src/components/ftbot/FTBotAPIPairList.vue

```diff
@@ -0,0 +1,82 @@
+<template>
```

Comment:
wouldn't a simple `Pairlist.vue` suffice (given that it's in the ftbot directory ...) ? 

otherwise, we might as well rename all components to `FTBotAPI*` - as they all work against the api one way or another ...

Reply:
We can name it whatever. I don't want to be the one to implement any style guide restrictions this early on because it might stifle rapid development -- I'll leave that up to you when you feel the time is right; however, this naming convention was produced with [this](https://vuejs.org/v2/style-guide/#Order-of-words-in-component-names-strongly-recommended) in mind.